<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassGenerationProfilesDialog</class>
 <widget class="QDialog" name="PassGenerationProfilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnNewProfile">
       <property name="text">
        <string>New...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteProfile">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listViewProfiles"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUseUppercase">
     <property name="text">
      <string>Uppercase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUseLowercase">
     <property name="text">
      <string>Lowercase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUseDigits">
     <property name="text">
      <string>Digits</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUseSymbols">
     <property name="text">
      <string>Symbols</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetSpecialSymbols" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
